### PR TITLE
Add collection icon to improve masonry and slideshow views.

### LINKED ISSUE #2408: Collection in Masonry and slide show view is not great
### Descriptive summary

In some of the different views for blacklight in sufia 7, the collection has wither the wrong icon or a very strange one...
### Expected behavior

I'm guessing triple boxes, but not entirely certain.  That would match with other collection views.
### Actual behavior

Slide show
screen shot 2016-08-08 at 2 05 05 pm

Tiles
screen shot 2016-08-08 at 2 05 37 pm
### Steps to reproduce the behavior

When I have some collections in your system
Then I search the system for nothing
And I choose Collection as object type
And I click on the Masonry and Slideshow views
Then I should see the normal collection icon
### Related work

Link to related tickets or prior related work here.

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #2408.

Shamelessly borrowed from psu-stewardship/scholarsphere#410. Thanks to @mtribone

@projecthydra/sufia-code-reviewers

### CODE REVIEW

File: app/assets/stylesheets/sufia/blacklight_gallery.scss [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* These styles are very specific to override the thumbnails from blacklight gallery */
+
+#documents .grid .document .thumbnail {
```
Comment (jcoyne) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: Please fix this @mjgiarlo

File: app/assets/stylesheets/sufia/blacklight_gallery.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  padding: 0;
+}
+#documents .grid .document .thumbnail > a > img {
```
Comment (jcoyne) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: No id selectors please.

Comment (jcoyne) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: No id selectors please.

Comment (mtribone) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: These will be tough because they are very specific and coming from Blacklight. I matched the declaration to the letter in order to override. Might need to do `html > body` instead of the ID selector. Big hammer.

Comment (jcoyne) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: Is there any reason not to fix blacklight-gallery?

Comment (mtribone) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: No. It exists in ScholarSphere in this way because we are focused on SS 3.0.

Comment (jcoyne) [inline app/assets/stylesheets/sufia/blacklight_gallery.scss]: https://github.com/projectblacklight/blacklight-gallery/pull/42

**Comment by mjgiarlo** [comment]
Removed the blacklight_gallery tweaks since that work has now gone into blacklight_gallery. I will include a bump to blacklight_gallery in this PR once a new version is released, or, alternatively, that can go into a follow-on PR.

**Comment by mjgiarlo** [comment]
@jcoyne re-review?

File: spec/indexers/sufia/collection_indexer_spec.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe Sufia::CollectionIndexer do
+  describe "fontawesome default icon" do
```
Comment (jcoyne) [inline spec/indexers/sufia/collection_indexer_spec.rb]: what does this have to do with fontawesome?

**Comment by jcoyne** [comment]
@mjgiarlo I didn't do any of the tweaks you did in blacklight gallery, I just removed the `#documents` css selector, so it would be easier to make your changes. blacklight-gallery 0.7.0 has been released.

File: spec/indexers/sufia/collection_indexer_spec.rb (hunk 3 above)
Comment (mtribone) [inline spec/indexers/sufia/collection_indexer_spec.rb]: The default collection icon is/was a font awesome icon. Perhaps a better describe would be "default collection icon?"

**Comment by mjgiarlo** [comment]
@jcoyne take three?
